WhoopTriggerzDesign/CustomViews/CustomSliderModifier.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  CustomSliderModifier.swift
+//  WhoopTriggerzDesign
+//
+//  Created by Jessica Perez on 5/10/23.
+//
+import SwiftUI
+
+struct CustomSliderModifier: ViewModifier {
+    //MARK: - Properties
+    @Binding var value : CGFloat
+    @State var isMuted: Bool = false
```

Comment:
I think we might need to make the `isMuted` property a binding as well because we might need to communicate the value of this button being pressed to the parent view.
```suggestion
    @Binding var isMuted: Bool
```

Reply:
Resolved in 1d0cfdf